config/server.go

```diff
@@ -449,6 +451,17 @@ type EthereumChainDriverConfig struct {
 	Connection string `toml:"connection" desc:"ethereum node connection url default: http://localhost:7545"`
 }
 
+type ReleaseChangesConfig struct {
+	DomainChange1 int64 `toml:"domainChange_1" desc:"changes related to purchase and update domain"`
+}
+
+func DefaultReleaseChangesConfig() *ReleaseChangesConfig {
+	var cfg ReleaseChangesConfig
+	cfg.DomainChange1 = 200000
```

Comment:
can we have it config.toml as 200000 and here it can default to 0 right?

Reply:
I think we can make 200k here, but change the devnet command to have it become 0 so that our test scripts works